debug.log

```diff
@@ -0,0 +1,80 @@
+# Logfile created on 2016-09-05 13:26:07 +1000 by logger.rb/47272
```

Comment:
Don't check in your debug.log. Be a lot more careful over checking what you are committing. Give yourself a code review for everything you are about to commit. And add debug.log to your .gitignore file.